### ISSUE TITLE
Season Reset Glitch

### ISSUE THREAD

**Issue by ProfessorOfK3ology** [Glitchfiend/SereneSeasons#347]
I've noticed a glitch with worlds over one seasonal cycle old.

If a world is over one seasonal cycle (full spring-winter cycle) old, going into another world, causing some sort of lag (like reloading resource packs), and going back to the original world causes the original world's cycle to reset back to early spring.

This bug also occurs when you create a new world and then add enough time to advance the cycle, though you don't have to change worlds for this to happen. I don't know if these issues are related, though.

Here are two videos of me going through the processes to cause the bug:
https://user-images.githubusercontent.com/88566391/128584935-6313f9b8-c807-4e21-bb62-474ce1a259d8.mp4
https://user-images.githubusercontent.com/88566391/128584937-bff923fd-16a2-4afc-add6-7f77c82f4faf.mp4
____
The version of Serene Seasons I'm using is 1.16.5-4.0.1.119, and the only other mods I have installed are Optifine 1.16.5 HD U G8, WorldEdit 7.2.5, and Simple World Timer 1.16.4-1.1.2. None of these seem to affect Serene Seasons or the actual world time itself, so I've ruled them out of causing this bug; however, correct me if I'm wrong.

Edit: Forgot to list Forge version, I'm using 36.1.0

**Comment by MAGGen-hub** [comment]
Same problem... But with better-end-forge mod... Always reset season after switching... Don't know why...

**Comment by Ceimnithe** [comment]
I noticed an issue like this as well. Was playing on a save that was sometime in winter, loaded a different save and came back - I was in early spring. Thought that was odd but figured I just hadn't been paying attention to the season progression. Went to another save again, came back, now its mid summer, so something is definitely up, especially since I set my subseason duration to 25 days lol. I haven't been playing on my current setup long, so I can't say if this was happening before a certain point...as far as I know its been fishy ever since I installed my current setup.

I'll attach my seasons.toml in .txt format if it helps.

SereneSeasons-1.16.5-4.0.1.121
Minecraft 1.16.5
Forge 1.16.5 - 36.2.0
[seasons.txt](https://github.com/Glitchfiend/SereneSeasons/files/7035252/seasons.txt)
